openedx/core/djangoapps/content/learning_sequences/docs/decisions/0002-extensions-to-inter-app-apis.rst

```diff
@@ -0,0 +1,80 @@
+Extensions to Inter-app API Conventions
```

Comment:
There are only a few small places in this ADR where you reference `learning_sequences` specifically, with a slight tweak in language, it'd be easily applicable to a those top-level inter-app API definitions you mentioned.

Reply:
Also, this all seems like great common sense suggestions on package organization.  I agree with our off-line discussion about how we may want to tweak the exact details/suggestions, but in general this feels like a no-brainer.  

The only thing I wonder is whether there are any downsides to turning everything into data classes instead of returning ORM models from the API.  I like the idea, but as you've said, I'm sure we'll find an exception to the rule eventually.

It may not belong in this ADR, but I wonder if there's guidance we can write down about what to do when you want to extend/work with query sets from a package's API.  🤔